nginx.agent.conf

```diff
@@ -36,15 +36,8 @@ http {
             proxy_redirect http://$http_host/ /pkgpanda/;
         }
 
-        location /system/logs/v1/ {
-            proxy_buffering off;
-            proxy_cache off;
-
-            # make event source work with nginx
-            # http://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx
-            proxy_set_header Connection '';
-            proxy_http_version 1.1;
-            chunked_transfer_encoding off;
+        location /system/v1/logs/v1/ {
+            include common/server-sent-events.conf;
 
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
Same goes for the metrics endpoint. Seems like the X-Real-IP and the like are good things to include.